package sp0427;

import java.sql.Connection;
import java.sql.SQLException;

import javax.sql.DataSource;

import org.springframework.context.support.ClassPathXmlApplicationContext;

public class Main {
	public static void main(String[] args) throws Exception {
		ClassPathXmlApplicationContext context = 
				new ClassPathXmlApplicationContext("applicationContext.xml");
		DataSource ds = context.getBean(DataSource.class);
		Connection con = ds.getConnection();
		System.out.println("DB연결성공");
	}
}
